src/test/java/edu/cornell/kfs/fp/batch/service/impl/ProcurementCardCreateDocumentServiceImplTest.java

```diff
@@ -26,7 +26,7 @@
     private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ProcurementCardSummaryFeedService.class);
     
     private UnitTestSqlDao unitTestSqlDao;
-	
```

Comment:
since this file has only formatting changes I don't think it should be included in this commit

Reply:
OK, I will see if I can this out of the comit